plugins/janus_audiobridge.c

```diff
@@ -2731,6 +2731,7 @@ static void *janus_audiobridge_handler(void *data) {
 			json_t *gain = json_object_get(root, "volume");
 			json_t *record = json_object_get(root, "record");
 			json_t *recfile = json_object_get(root, "filename");
+			json_t *display = json_object_get(root, "display");
```

Comment:
You need to add display to `configure_parameters` as well, for automatic JSON validation.